main.go

```diff
@@ -61,7 +63,7 @@ func main() {
 
 	// Create a Kubernetes client.
 	// Initialize a configuration based on the default service account.
```

Comment:
Nit pick comment. It would not be default service account if a custom kubeconfig is passed in.